tests/integration_tests/modules/test_combined.py

```diff
@@ -74,23 +74,16 @@ def test_final_message(self, class_client: IntegrationInstance):
         """Test that final_message module works as expected.
 
         Also tests LP 1511485: final_message is silent.
-
-        It's possible that if this test is run within a minute or so of
-        midnight that we'll see a failure because the day in the logs
-        is different from the day specified in the test definition.
         """
         client = class_client
         log = client.read_from_file('/var/log/cloud-init.log')
-        # Get date on host rather than locally as our host could be in a
-        # wildly different timezone (or more likely recording UTC)
-        today = client.execute('date "+%a, %d %b %Y"')
         expected = (
             'This is my final message!\n'
             r'\d+\.\d+.*\n'
-            '{}.*\n'
+            '.*\n'  # Checking date is too error prone due to timezone
```

Comment:
Can we instead make our regex a bit more flexible?
```suggestion
           r'\w{3}, \d{2} \w{3} \d{4}'
```

Reply:
Yes, this is much better, thanks. The line includes time so I included that as well.